{
  "id": "EUVD-2019-0029",
  "enisaUuid": "3c4b3d61-6489-3bfd-bb7d-8f1d1bf20746",
  "description": "The Serialize.deserialize() method in CoAPthon3 1.0 and 1.0.1 mishandles certain exceptions, leading to a denial of service in applications that use this library (e.g., the standard CoAP server, CoAP client, example collect CoAP server and client) when they receive crafted CoAP messages.",
  "datePublished": "Apr 2, 2019, 7:29:00 PM",
  "dateUpdated": "Aug 25, 2021, 4:57:20 AM",
  "baseScore": 7.5,
  "baseScoreVersion": "3.0",
  "baseScoreVector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
  "references": [
    "https://github.com/Tanganelli/CoAPthon3/issues/16",
    "https://github.com/advisories/GHSA-c6fm-rgw4-8q73",
    "https://nvd.nist.gov/vuln/detail/CVE-2018-12679",
    "https://github.com/pypa/advisory-database/tree/main/vulns/coapthon3/PYSEC-2019-166.yaml"
  ],
  "aliases": [
    "CVE-2018-12679",
    "GHSA-c6fm-rgw4-8q73",
    "PYSEC-2019-166"
  ],
  "assigner": "mitre",
  "epss": 0.37,
  "enisaIdProduct": [
    {
      "id": "4eb0248b-01b7-3d88-9bb4-34378fd8f4b4",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "d9553cd5-c0c1-3e3d-b484-22d48a3ca98c",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2018-12679"
}